compiler/src/parsing/parsetree_iter.re

```diff
@@ -3,8 +3,6 @@ open Parsetree;
 type hooks = {
   enter_location: Location.t => unit,
   leave_location: Location.t => unit,
-  enter_attribute: (attribute, attribute_context) => unit,
-  leave_attribute: (attribute, attribute_context) => unit,
```

Comment:
I don't like that you removed these. We might want to iter attributes at some point